pupa/tests/importers/test_event_importer.py

```diff
@@ -1,8 +1,7 @@
 import pytest
 from pupa.scrape import Event as ScrapeEvent
-from pupa.importers import EventImporter
-from opencivicdata.models import Jurisdiction
-
+from pupa.importers import EventImporter, OrganizationImporter, PersonImporter, BillImporter, VoteEventImporter
```

Comment:
Please update the test to actually test the import of participants (person and org) and related entities (person, org, bill, vote event). Such tests would pick up the typo in vote_event I commented on.